### PR TITLE
fix: upgrade shellbar to fundamental-styles 0.6.0

### PR DESCRIPTION
**Pull request by mikerodonnell89**
#### Please provide a link to the associated issue.

#1852  

#### Please provide a brief summary of this pull request.

Upgrade shellbar to 0.6.0

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [ ] update `README.md`
- [ ] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [ ] Documentation Examples
- [ ] Stackblitz works for all examples

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-02-25 at 2 19 12 PM
Is it like this in localhost or it's Netlify bug?

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-02-25 at 2 20 03 PM

In this example clicking on Corporate Portal does not open the dropdown menu

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-02-25 at 2 20 39 PM

I know it's not ShellBar related but the dropdown looks a bit broken, maybe remove the icons or add different icons for all list items

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-02-25 at 2 26 11 PM

Again not really related to your PR but maybe we can improve the content section. It looks like it's part of the SideNav. You can create a separate issue if you don't want to overload this PR

**Comment by mikerodonnell89** [comment]
> Screen Shot 2020-02-25 at 2 19 12 PM
> 
> Is it like this in localhost or it's Netlify bug?

It's fine on localhost, so I think it's a netlify thing

**Comment by JKMarkowski** [comment]
Hello @mikerodonnell89 I think there are problems with `ngClass` usage with `AbstractFdNgxClass`, or even `CssClassBuilder`.

File: libs/core/src/lib/combobox/combobox.component.ts [hunk 1]
```diff
@@ -234,6 +238,8 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
             .subscribe(index => this.onMenuClickHandler(index));
         this.menuKeyboardService.focusEscapeBeforeList = () => this.searchInputElement.nativeElement.focus();
         this.menuKeyboardService.focusEscapeAfterList = () => { };
+
+        this.searchInputElement.nativeElement.classList.add('fd-shellbar__input-group__input');
```
Comment (JKMarkowski) [inline libs/core/src/lib/combobox/combobox.component.ts]: Could you add some statement? It shouldn't be always applied

File: libs/core/src/lib/shellbar/shellbar.component.ts [hunk 2]
```diff
@@ -12,5 +21,23 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class ShellbarComponent {
+export class ShellbarComponent implements AfterContentInit {
+
+    @ContentChild(ComboboxComponent, {static: false})
```
Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar.component.ts]: Could You add `/** @hidden */` comment?

File: libs/core/src/lib/shellbar/shellbar.component.ts [hunk 3]
```diff
@@ -12,5 +21,23 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
...
+    comboboxComponent: ComboboxComponent;
+
+    @ContentChildren(forwardRef(() => ButtonComponent))
```
Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar.component.ts]: Could You add `/** @hidden */` comment?

File: libs/core/src/lib/shellbar/shellbar.component.ts [hunk 4]
```diff
@@ -12,5 +21,23 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
...
+    buttons: QueryList<ButtonComponent>;
+
+    ngAfterContentInit() {
```
Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar.component.ts]: Could You add `/** @hidden */` comment? + Type returned

Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar.component.ts]: Maybe let's use

```suggestion
    ngAfterContentInit() {
        this.applyShellbarModeToCombobox();
        this.applyShellbarModeToButtons();
    }
```

File: libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.html [hunk 5]
```diff
@@ -56,5 +54,6 @@
     </ng-container>
 </ng-container>
 
-
-<ng-content select="fd-product-switch"></ng-content>
+<div [ngClass]="{'fd-shellbar__action': productSwitchComponent}">
```
Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.html]: Maybe consider using
```suggestion
<div class="fd-shellbar__action" *ngIf="productSwitchComponent">
```

File: libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions-mobile.component.ts [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+    /** @hidden */
+    toggleCollapsedProducts(event: MouseEvent): void {
+        event.preventDefault();
+        event.stopPropagation();
+        this.showCollapsedProducts = !this.showCollapsedProducts;
+    }
+
+    /** @hidden */
+    ngAfterContentChecked() {
```
Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions-mobile.component.ts]: Missin return type

File: libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.ts [hunk 7]
```diff
@@ -83,7 +80,8 @@ export class ShellbarActionsComponent implements OnInit, AfterContentChecked {
     comboboxComponent: ComboboxComponent;
 
     /** @hidden */
-    totalNotifications: number;
+    @ContentChild(ProductSwitchComponent, {static: false})
+    productSwitchComponent: ProductSwitchComponent;
```
Comment (JKMarkowski) [inline libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.ts]: in `95` line there is `actionClicked` method which is unused, let's remove it

**Pull request closed without merging**